pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+line-length = 79
```

Comment:
```suggestion
line-length = 88
```

How about 88?

Reply:
Sounds good, I updated the line length settings everywhere, so it's consistently 88 now. So after re-running Black on all files, this means there are far fewer updates, the only ones are pretty much just the trailing commas which Black adds now that it's set to support Python 3.6+ only (which happens to be the same as Relé as far as I understood).